### ISSUE TITLE
503 - User over quota.

### ISSUE THREAD

**Issue by seaquake** [jdgeenen/gcontactsync#92]
Hi,
always more often i received the error of "token expired" and the "503 temporary problem - please try again later and consider using batch operations. The user over quota".

Thunderbird 38.3.0
gcontactsync 2.0.7

If there is some log that i can send to you, let me know.
![cattura](https://cloud.githubusercontent.com/assets/1249550/10375104/ef81cf00-6df6-11e5-94e2-63ab13cfa3f1.JPG)

Thanks for support

**Comment by jdgeenen** [comment]
Could you go to gContactSync -> Preferences -> Advanced and click on "Run Throttle Test" and let me know what it says?

**Comment by seaquake** [comment]
I have run the test from 5 mins, after i received the "503 error" and seems that the test is stuck on "Testing with a delay of: 0 ms". i let it run, if nothing change i will let pass sometime and try it when not in the "over quota" period.

![cattura](https://cloud.githubusercontent.com/assets/1249550/12222079/6865ed4e-b7b1-11e5-9bad-b1d090220726.JPG)

At the moment i'm runnning THunderbird 38.5.1 and gcontactsync 3.0.0a1

Thanks for support

**Comment by seaquake** [comment]
i'm sorry i just retry (after 2 hours with thunderbird closed) butnothing change.

**Comment by jdgeenen** [comment]
Were there some 401 errors in the log?  I saw the same thing when I tried it so I wrote up Issue #95 to look into it.

**Comment by seaquake** [comment]
No, i try a search in the log after a stucked test, but there isn't any 401.

**Comment by jdgeenen** [comment]
Could you turn on Verbose logging in the Preferences and e-mail it to joshgeenen at gmail?

**Comment by seaquake** [comment]
Done, thanks.

**Comment by jonnybest** [comment]
I seem to have this issue after my initial synchronization with Google Contacts. After ~2000 items, the "throttle test" gets nothing but 401 errors. Unfortunately, gcontactsync only tells me "Your authentication token from Google has expired. (...)". Is this the same issue?

**Comment by seaquake** [comment]
I receive that error too, but i don't know if it are connected. for sure there is a limit the google set on the request. i'm still waiting the fix of this issue,
we need wait that author fix it :D
